test/integration/device-api/v1.spec.ts

```diff
@@ -0,0 +1,41 @@
+import * as express from 'express';
+import { SinonStub, stub } from 'sinon';
+import * as request from 'supertest';
+
+import * as config from '~/src/config';
+import * as deviceApi from '~/src/device-api';
+import * as actions from '~/src/device-api/actions';
+import * as v1 from '~/src/device-api/v1';
+
+describe('device-api/v1', () => {
+	let api: express.Application;
+
+	before(async () => {
+		await config.initialized();
+
+		// `api` is a private property on SupervisorAPI but
+		// passing it directly to supertest is easier than
+		// setting up an API listen port & timeout
+		api = new deviceApi.SupervisorAPI({
+			routers: [v1.router],
+			healthchecks: [],
+			// @ts-expect-error
+		}).api;
+	});
+
+	describe('GET /v1/healthy', () => {
+		// Actions are tested elsewhere so we can stub the dependency here
+		beforeEach(() => stub(actions, 'runHealthchecks'));
```

Comment:
Instead of stubbing `actions.runHealthchecks` which is an implementation detail from the point of view of the API user, you could pass stubs when initializing the API

```
api = new deviceApi.SupervisorAPI({
    routers: [v1.router],
    healthchecks: [stub().resolve(true), stub.resolve(false)],
	// @ts-expect-error
    }).api
    
await request(api).get('/v1/healthy').expect(200);
```

Reply:
Done in <TODO>